features/tab-access-tabs.feature

```diff
@@ -21,6 +21,19 @@ Feature: Adjust tab stops
 
 
   @wip
+  Scenario Outline: Get tab stop values
+    Given a tab_stops having 3 tab stops
+     Then the tab stop at index <index> has position <position>
+      And the tab stop at index <index> has alignment <alignment>
+      And the tab stop at index <index> has leader <leader>
+     
+    Examples: tab stop positions
+      | index | position | alignment              | leader                  |
+      | 0     | 914400   | WD_TAB_ALIGNMENT.LEFT  | WD_TAB_LEADER.SPACES |
+      | 2     | 2743200  | WD_TAB_ALIGNMENT.RIGHT | WD_TAB_LEADER.DOTS   |
```

Comment:
The short story on this one is to bring it down to operating on an individual tab stop, and second, to test each property separately.

By combining TabStops with TabStop here you're actually testing two things at once; TabStops.__getitem__() and TabStop.position (or .alignment, or .leader).

By combining all the properties together, you force testing combinations, which are more complex without a distinct benefit.

So I'd be seeing this divided into three separate tests like this:

```cucumber
@wip
Scenario Outline: Get tab stop position
  Given a tab stop having position <position>
   Then tab_stop.position is <value>

  Examples: tab stop positions
    | position | value  |
    | 914400   | 914400 |
    | other cases as necessary to drive out corner cases (only if necessary, otherwise this becomes a single scenario rather than a scenario outline). If there's no case where value is different than position, you can just use the single position column.
```

The similar for alignment and leader.